framework/src/modules/chain/submodules/blocks/chain.js

```diff
@@ -64,11 +69,13 @@ class Chain {
 			balancesSequence,
 			logic: {
 				block,
-				initTransaction,
 			},
 			channel,
 		};
 		self = this;
+		self.modules = {
```

Comment:
Can we assign interfaceAdapters to a different variable name. Lest we will have `self.modules` and `module` variables which have different values.

Reply:
true, but this will be changed in the coming block PR